python/ray/train/tests/test_lightning_trainer.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+from ray.train.lightning import LightningConfigBuilder, LightningTrainer
+import ray
+from ray.air.util.data_batch_conversion import convert_batch_type_to_pandas
+import pytest
+
+from ray.train.tests._lightning_utils import (
+    LinearModule,
+    DoubleLinearModule,
+    DummyDataModule,
+)
+
+
+@pytest.mark.parametrize("accelerator", ["cpu", "gpu"])
+@pytest.mark.parametrize("datasource", ["dataloader", "datamodule"])
+def test_trainer_with_native_dataloader(accelerator, datasource):
```

Comment:
Can we add `ray.init/ray.shutdown` fixtures for each test? Can look for `ray_start_x_cpus` in other tests.